Modules/process/process_processlist.php

```diff
@@ -155,6 +155,8 @@ public function core_process_list()
         $list[57] = array(_(" * source feed"),ProcessArg::FEEDID,"multiply_by_source_feed",0,DataType::UNDEFINED,"Virtual", 'desc'=>"");
         $list[58] = array(_(" / source feed"),ProcessArg::FEEDID,"divide_by_source_feed",0,DataType::UNDEFINED,"Virtual", 'desc'=>"");
         $list[59] = array(_("1/ source feed"),ProcessArg::FEEDID,"reciprocal_by_source_feed",0,DataType::UNDEFINED,"Virtual", 'desc'=>"");
+        
+        $list[60] = array(_("Reset to ONE"),ProcessArg::NONE,"reset2one",0,DataType::UNDEFINED,"Misc", 'desc'=>"The value '1' is passed back for further processing by the next processor in the processing list.");
```

Comment:
Dont agree on this, 'one' is not a special number different from any other than zero. So you can get the same behaviour for all numbers by setting to 'zero' and adding a fixed number with process '+' after.